### PR TITLE
AIDA-1480: Add VideoJustification

### PR DESCRIPTION
**Pull request by ecurley-nextcentury**
Aside from adding the VideoJustification, this PR makes changes to the way java ontology constants are created and removes AidaAnnotationOntology.java in favor of InterchangeOntology.java.

@phile-caci, when you get a chance to take a look at this, let me know an I can walk through it with you.

### CODE REVIEW

File: java/src/main/java/com/ncc/aif/AIFUtils.java [hunk 1]
```diff
@@ -995,6 +995,52 @@ public static Resource markAudioJustification(Model model, Collection<Resource>
         return justification;
     }
 
+    /**
```
Comment (phile-caci) [inline java/src/main/java/com/ncc/aif/AIFUtils.java:998]: @ecurley-nextcentury ,

Do we need a markVideoJustification also?

File: java/src/test/java/com/ncc/aif/ExamplesAndValidationTest.java [hunk 2]
```diff
@@ -1310,6 +1313,22 @@ void audioJustification() {
...
+            /**
+             * Create video justifications with and without optional URIs.  Without a URI, a blank node is created.
+             */
+            @Test
+            void videoJustification() {
+                final Double startTimestamp = 5.0;
+                final Double endTimestamp = 10.0;
+
+                Resource justification = makeVideoJustification(model, utils.getDocumentName(), startTimestamp, endTimestamp, InterchangeOntology.VideoJustificationChannelBoth, system, confidence);
+                makeVideoJustification(model, utils.getDocumentName(), startTimestamp*1.1, endTimestamp*1.1, InterchangeOntology.VideoJustificationChannelSound, system, confidence, utils.getUri("custom-uri-" + ++uriCount));
```
Comment (phile-caci) [inline java/src/test/java/com/ncc/aif/ExamplesAndValidationTest.java:1326]: @ecurley-nextcentury,

Just curious, what does the *1.1 do  in "startTimestamp*1.1, endTimestamp*1.1,"

**Comment by phile-caci** [comment]
@ecurley-nextcentury ,

Just some comments on the code. I was able to setup, build and ran mvn test. Everything else looks good to me. Thanks!

File: java/src/main/java/com/ncc/aif/AIFUtils.java (hunk 1 above)
Comment (ecurley-nextcentury) [reply java/src/main/java/com/ncc/aif/AIFUtils.java:998]: The mark...() APIs are difficult to maintain and don't save anyone much. All the parameters for make...() are still required and it saves a single call to markJustification. Let me know if that makes sense to you. I'm happy to hear counterpoints.

File: java/src/test/java/com/ncc/aif/ExamplesAndValidationTest.java (hunk 2 above)
Comment (ecurley-nextcentury) [reply java/src/test/java/com/ncc/aif/ExamplesAndValidationTest.java:1326]: Haha, nothing. Good catch. I was copying/pasting from audioJustification(). I'll fix it and push a change to remove that.

**Comment by ecurley-nextcentury** [comment]
> Was the python changes covered by this ticket as well? Thanks!

The ticket does cover them, but in order to get the validator out earlier, Craig outlined a release where only java and the validator are covered. I will dupe 1480 and add it to a later release of AIF.

File: java/src/test/java/com/ncc/aif/ExamplesAndValidationTest.java (hunk 2 above)
Comment (ecurley-nextcentury) [reply java/src/test/java/com/ncc/aif/ExamplesAndValidationTest.java:1326]: Oops, looks like I did use the timestamps to create multiple justifications. The purpose behind multiplying by 1.1 is to make sure the justifications aren't all pointing to the same locations. Now that I realize that I _did_ actually make multiple justifications, I'm not going to make any change. Sorry for the confusion

**Comment by ecurley-nextcentury** [comment]
> I will dupe 1480 and add it to a later release of AIF.

The cloned ticket is [1492](https://nextcentury.atlassian.net/browse/AIDA-1492)

**Pull request merged**